cli/bin/internal_command_update_prompt.dart

```diff
@@ -42,5 +42,7 @@ class InternalCommandUpdatePrompt extends InternalCommand {
   @override
   void run(final List<String> args, Context context) {
     shouldAutoUpdateAfterExecutingCommand(args, context);
+    context.printToConsole(
```

Comment:
I changed my mind—this message is no longer needed here. But you need to remove the usage of the `InternalCommandUpdatePrompt.shouldAutoUpdateAfterExecutingCommand` so we would not ask users to update the tool for now. Until we onboard Stax with Brew and Chocolatey fully, and we will be able to check what the current version the user is using and what the most recent one is on the server.